app/lib/removeColumns.js

```diff
@@ -35,10 +35,9 @@ function removeColumns() {
       const pomiDataReader = fs.createReadStream(`${OUTPUT_DIR}/${POMI_FILE}`);
       const reducedPomiDataWriter =
         fs.createWriteStream(`${OUTPUT_DIR}/${REDUCED_POMI_FILE}`);
-      const parser = parse({ from: 2 });
 
       pomiDataReader
-        .pipe(parser)
+        .pipe(parse())
```

Comment:
this section looks a bit odd as the pipes mix function calls and function names, but I think this is down to the way the csv library is implemented, so probably can't be helped.